apps/coscrad-frontend/src/components/resources/songs/song-detail.container.tsx

```diff
@@ -0,0 +1,12 @@
+import { useLoadableSongById } from '../../../store/slices/resources';
+import { displayLoadableSearchResult } from '../../higher-order-components/display-loadable-search-result';
+import { SongDetailPresenter } from './song-detail.presenter';
+
+export const SongDetailContainer = (): JSX.Element => {
+    const loadableSong = useLoadableSongById();
+
+    // Huge gotcha- if we accidentally use `displayLoadableWithErrorsAndLoading` here!
```

Comment:
This may be an argument for having a higher order component for ResourceIndex and ResourceDetail.